src/test/java/de/tum/cit/aet/artemis/architecture/ResourceArchitectureTest.java

```diff
@@ -34,6 +35,8 @@
 
 class ResourceArchitectureTest extends AbstractArchitectureTest {
 
+    private static final Pattern KEBAB_CASE_PATH_PATTERN = Pattern.compile("^(\\.?[a-z0-9]+(-[a-z0-9]+)*|\\{[^}]+})(/(([a-z0-9]+(-[a-z0-9]+)*|\\{[^}]+})))*(.json|/\\*)?$");
```

Comment:
Regex makes sense.

One small question: I assume the `.json` at the end means exactly ".json" for JSON accesses. Don't we have to escape the dot, then?

Otherwise, we'd match strings like

```
foo/barXjson
```

Reply:
good point, that's true. I'll fix that